backend/src/errors.rs

```diff
@@ -0,0 +1,13 @@
+error_chain! {
+    links {
+    }
+
+    foreign_links {
+        ::std::io::Error, Io, "io error";
+        ::json::Error, Json, "json error";
+        ::chrono::ParseError, Chrono, "chrono parsing error";
```

Comment:
We can land it and see if it pulls its weight. I've not used it before so I'm interested to see how it pans out.